src/mode/modeHandler.ts

```diff
@@ -280,10 +295,11 @@ export class SearchState {
     }
   }
 
-  constructor(direction: SearchDirection, startPosition: Position, searchString = "") {
+  constructor(direction: SearchDirection, startPosition: Position, searchString = "", { isRegex = false } = {}) {
```

Comment:
This is actually pure ES6 JS syntax - it's parameter destructuring. I would say to do it without the default object assignment, so just `{ isRegex = false }`